implants/lib/transport/Cargo.toml

```diff
@@ -27,6 +29,9 @@ hyper = { version = "0.14", features = [
 ] } # Had to user an older version of hyper to support hyper-proxy
 hyper-proxy = {version = "0.9.1", default-features = false, features = ["rustls"]}
 hickory-resolver = { version = "0.24", features = ["dns-over-https-rustls", "webpki-roots"], optional = true }
+base32 = { version = "0.5", optional = true }
```

Comment:
Move these to the workspace root and use `= { workspace = true }` this helps managing shared dep versions across the different components and reduces agent bloat.